### PR TITLE
adopt pytest as test runner

### PR DESCRIPTION
**Pull request by saxix**

### CODE REVIEW

File: src/etools/applications/partners/admin.py [hunk 1]
```diff
@@ -9,11 +9,11 @@
 
 from etools.applications.EquiTrack.admin import ActivityInline, SnapshotModelAdmin
 from etools.applications.partners.exports import PartnerExport
-from etools.applications.partners.forms import (PartnersAdminForm,  # TODO intervention sector locations cleanup
-                                                PartnerStaffMemberForm, SectorLocationForm,)
+from etools.applications.partners.forms import PartnersAdminForm  # TODO intervention sector locations cleanup
```
Comment (reinbach) [inline src/etools/applications/partners/admin.py]: How come you're separating the import with a comment on it from the others?

File: src/etools/libraries/utils/__init__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from inspect import isclass
+
+
+def fqn(o):
+    """Returns the fully qualified class name of an object or a class
```
Comment (dpoirier) [inline src/etools/libraries/utils/__init__.py]: I like this. I always worry about typos when we embed code references into strings.

File: src/etools/applications/partners/admin.py (hunk 1 above)
Comment (saxix) [reply src/etools/applications/partners/admin.py]: it is a script, need some cleanup later

File: src/etools/libraries/utils/__init__.py (hunk 2 above)
Comment (saxix) [reply src/etools/libraries/utils/__init__.py]: :)

**Pull request closed without merging**